### PR TITLE
documentation of subset of parameters for display

### PR DESCRIPTION
**Pull request by rtmatx**
Hi,

I have updated the docs to show the approach for selecting subset of parameters for display. This is very minor update. I haven't used `jupyter notebook` style documentation generation before. So I am sure, what I should do other than editing the appropriate `.ipynb` file. I can extend the documentation for this method in this PR. But I just want to start this PR and take your feedback to proceed to next steps

Thanks
Ravi

### CODE REVIEW

**Comment by philippjfr** [comment]
Thanks, could you delete the checkpoint notebook and clear the notebook before committing it?

**Comment by rtmatx** [comment]
I have cleared the outputs and removed the checkpoint files

**Comment by rtmatx** [comment]
I have few extensions that is making unnecessary changes to the notebook. I will clear them and make a new commit

**Comment by philippjfr** [comment]
Sorry for the hassle but your notebook configuration seems to be adding a ton of metadata to the notebook. Could you run this bash script against the notebook to clear them?

```
#!/bin/bash
jq --indent 1 \
    '
    (.cells[] | select(has("outputs")) | .outputs) = []
    | (.cells[] | select(has("execution_count")) | .execution_count) = null
    | .metadata = {"language_info": {"name":"python", "pygments_lexer": "ipython3"}}
    | .cells[].metadata = {}
    ' $1 > /tmp/stripped.ipynb
```

It'll copy a cleared copy of the notebook to /tmp/stripped.ipynb

File: examples/user_guide/Param.ipynb [hunk 1]
```diff
@@ -382,6 +398,29 @@
...
+    "    | .cells[].metadata = {}\n",
+    "    ' $1 > ~/Desktop/Param.ipynb"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": []
```
Comment (philippjfr) [inline examples/user_guide/Param.ipynb]: Looking good now, except for this.

**Comment by philippjfr** [comment]
It looks like you deleted the notebook this time, sorry this is turning out to be such a headache.

**Comment by philippjfr** [comment]
It looks like there's still lots of metadata in there, I'm happy to merge this and clear it in a separate commit myself, but it would be good to figure out why you're having such issues with the clearing bash script.

**Comment by rtmatx** [comment]
I am sorry for all the commits. I am just figuring out how to commit notebooks. Never done this before. I installed the jq and ran the command that you shared. It worked out smoothly except at few places which I think is an issue that is specific to me. 

Let me know if there is still some unnecessary info in the notebook file.

I am sorry for the trouble. This small PR shouldn't have taken so long

**Comment by philippjfr** [comment]
Oh don't worry about the commits, we generally squash merge anyway. Sounds like we need to adapt the script a bit to make sure it also clears the metadata that you were seeing. Anyway thanks a lot and sorry for all the hassle!

**Comment by rtmatx** [comment]
No problem. Did the latest commit remove all the metadata?.

Thanks a lot for all the libraries. I am using them extensively and wouldn't mind making small improvements that I can. Are there any beginner friendly issues that I can work on in holoviews/datashader/hvplot/panel/param eco-system. I will learn my way through if it doesn't need lot of time commitment for the beginning

**Comment by philippjfr** [comment]
If you are available to fill gaps in panel docs that would be a huge help already. We're in the final stretch before we officially announce the library and it would be good to get the docs in a more solid shape before then.

File: examples/user_guide/Param.ipynb [hunk 2]
```diff
@@ -192,6 +192,22 @@
    "source": [
     "The parameterized and annotated ``view`` method could return any one of the types handled by the [Pane objects](./Panes.ipynb) panel provides, making it easy to link parameters and their associated widgets to a plot or other output. Parameterized classes can therefore be a very useful pattern for encapsulating a part of a computational workflow with an associated visualization, declaratively expressing the dependencies between the parameters and the computation.\n",
     "\n",
+    "`pn.panel` can be used to use only a subset of parameters. For example, the code below uses only `phase`  while maintaining `sine.frequency` at value `1`"
```
Comment (jbednar) [inline examples/user_guide/Param.ipynb]:
```suggestion
    "By default, a Parameters pane will show all parameters with a `precedence` value above the value `pn.Param.display_threshold`, so you can use `precedence` to automatically hide parameters that are not meant to have widgets.  You can also explicitly choose which parameters should have widgets in a given pane, by passing a `parameters` argument.  For example, the code gives a `phase`  widget, while maintaining `sine.frequency` at the initial value of `1`"
```

**Comment by philippjfr** [comment]
If you're happy to help out on that front I'd be happy to make you an official contributor so you can push branches to the main repo and we can collaborate on PRs more easily.

**Comment by jbednar** [comment]
Thanks for the contribution; it's very helpful for us to get feedback about specific topics that are not clear and suggestions for how to make them clearer!

File: examples/user_guide/Param.ipynb (hunk 2 above)
Comment (rtmatx) [reply examples/user_guide/Param.ipynb]: That looks more readable and less confusing

Comment (jbednar) [reply examples/user_guide/Param.ipynb]: Glad to hear it; please commit the suggestion if it's ok (I don't have permissions to push to your fork).

**Comment by rtmatx** [comment]
> If you're happy to help out on that front I'd be happy to make you an official contributor so you can push branches to the main repo and we can collaborate on PRs more easily.

Sure. I can contribute. I actually used `panel` in the past week. I recently wrote a script for exploring a dataframe using panel internals. The script is available at https://github.com/rtmatx/scripts/blob/master/panel_dashboard.py

**Comment by philippjfr** [comment]
Cool, I'll try that out. You should have received a collaborator invite.

**Comment by rtmatx** [comment]
> If you're happy to help out on that front I'd be happy to make you an official contributor so you can push branches to the main repo and we can collaborate on PRs more easily.

Sure. I have used

**Comment by rtmatx** [comment]
> Cool, I'll try that out. You should have received a collaborator invite.

Thanks. I accepted the invitation. I will keep this PR open until it is merged. I am available on pyviz gitter channel under the same alias as my current github. Let me know, what is a good place to start with. Thanks

File: examples/user_guide/Param.ipynb (hunk 1 above)
Comment (rtmatx) [reply examples/user_guide/Param.ipynb]: I don't see this in the current commit. I am assuming, this got fixed somehow

**Comment by rtmatx** [comment]
> Thanks for the contribution; it's very helpful for us to get feedback about specific topics that are not clear and suggestions for how to make them clearer!

Thank you @jbednar. I will provide feedback and contribute in whatever way I can.

**Comment by philippjfr** [comment]
Thanks again!

**Pull request merged**